### ISSUE TITLE
windows下运行inference.py出现编码问题

### ISSUE THREAD

**Issue by baiyueying** [anliyuan/Ultralight-Digital-Human#24]
请问大佬，当我运行inference.py的时候，他爆出以下问题
OpenCV: FFMPEG: tag 0x47504a4d/'MJPG' is not supported with codec id 7 and format 'mp4 / MP4 (MPEG-4 Part 14)'
OpenCV: FFMPEG: fallback to use tag 0x7634706d/'mp4v'
D:\xx\miniconda3\envs\dh\lib\site-packages\torch\nn\functional.py:1967: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.
  warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.")
这是否需要修改编码格式，感谢

**Comment by anliyuan** [comment]
> 请问大佬，当我运行inference.py的时候，他爆出以下问题 OpenCV: FFMPEG: tag 0x47504a4d/'MJPG' is not supported with codec id 7 and format 'mp4 / MP4 (MPEG-4 Part 14)' OpenCV: FFMPEG: fallback to use tag 0x7634706d/'mp4v' D:\xx\miniconda3\envs\dh\lib\site-packages\torch\nn\functional.py:1967: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead. warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.") 这是否需要修改编码格式，感谢

应该是 改一下试试。

**Comment by SJseventeen** [comment]
> 请问大佬，当我运行inference.py的时候，他爆出以下问题 OpenCV: FFMPEG: tag 0x47504a4d/'MJPG' is not supported with codec id 7 and format 'mp4 / MP4 (MPEG-4 Part 14)' OpenCV: FFMPEG: fallback to use tag 0x7634706d/'mp4v' D:\xx\miniconda3\envs\dh\lib\site-packages\torch\nn\functional.py:1967: UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead. warnings.warn("nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.") 这是否需要修改编码格式，感谢
请问这三个报错是如何处理的，尤其是最后一个

**Comment by tangyong2000** [comment]
为什么我win10装不上conda install mkl=2024.0
